google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/spi/v1/WatchdogInterceptorTest.java

```diff
@@ -1,202 +0,0 @@
-/*
```

Comment:
Can this test be refactored to test new functionality ?

Reply:
I thought about that, but these tests are all unit tests with mocks and the [test](https://github.com/googleapis/gax-java/blob/master/gax/src/test/java/com/google/api/gax/rpc/WatchdogTest.java) for watchdog in gax is mostly equivalent so it seems unnecessary.